src/extension/extensionServer.ts

```diff
@@ -240,8 +240,15 @@ export class ExtensionServer implements vscode.Disposable {
                         return mobilePlatform.runApp();
                     })
                     .then(() => {
+                        if (request.arguments.type === "reactnativedirect") {
+                            generator.step("mobilePlatform.enableDirectDebuggingMode");
+                            generator.add("isDirect", true, false);
+                            this.logger.info(localize("PrepareHermesDebugging", "Prepare Hermes Debugging"));
```

Comment:
```suggestion
                            this.logger.info(localize("PrepareDirectDebugging", "Prepare direct debugging"));
```

Reply:
I suppose that `Direct` is our internal name for the debugger, but the name `Hermes` could be more clear for users.

Reply:
Yes, but in future the same debugger will be used for debugging React Native Windows applications and having "Hermes" mentions in logs will be more confusing. Anyway, this is just logs. We have hermes debug configurations and it will be enough for customer, I guess.

Reply:
I think we have two ways here:
1) Change this message to one general message like `Prepare direct debugging`
2) Leave it as is but add strict division by `platform`. For example for `platform` Android it will be `Prepare Hermes debugging`.
My concern is that `Prepare direct debugging` message might not obvious for customers.

Reply:
I separated the log message for Hermes debugging and provided an opportunity to add a new log message for React Native Windows debugging afterward.